docs/3_setup_guide/Delft_Fiat/hydromt_fiat_gui.qmd

```diff
@@ -0,0 +1,229 @@
+
+# Setup Guide
+## Installation
+To run the Floodadapt model builder you need to install two tools:
+
+  1. **HydroMT FIAT**
+  2. **DelftDashboard**
+
+#### HydroMT FIAT
+HydroMT-FIAT is available from pypi and will be added to conda-forge (in progress).
+
+To install hydromt_fiat for usage, do:
+
+  > pip install hydromt_fiat
+
+#### DelftDashboard
+For the Graphical User Interface you additionally need to set up **DelftDashboard**. You can find the installation guide [here](https://github.com/Deltares-research/DelftDashboard/blob/FloodAdapt_ModelBuilders/README.md).
+
+# User Guide
+After you open the GUI you must select a model output directory before you start.
+
+*GUI instructions*
+  
+  1. Go to **File** >> **Select Working Directory**.
+  2. Select an output folder.
+
+Now you are ready to set up the model.
+
+After creating your model you should end up with a folder structure as describted in the section [Data Input - Folder Structure](data_input.qmd).
+
+## Create Model Boundary
+
+There are multiple options on how to create the model boundaries:
+
+  * Draw polygon/ draw box  
+  * SFINCS model domain 
+  * Upload file (polygon vector file e.g. *.gpkg*, *.geojson*)
+
+![Workflow to create model boundaries.](../../_static/figures/Draw_boundaries.svg){#fig-boundaries}
+
+*GUI instructions*
+
+  1. Go to the **Model Boundary | Quickbuild** -tab and select any of the options in the drop-down menu to create the boundary (@fig-boundaries).
+      a) **Draw polygon**: Click on the map and draw a valid polygon. Double click will end the drawing process.
+      b) **Draw box**: Click on the map and draw a box. Double click will end the drawing process.
+      c) **SFINCS model domain**: If you aleady created a SFINCS model you can select the **/gis-folder**. Do <u>not</u> select the region file itself. It will be recognized automatically by the model.
+      d) **Upload file**: You can upload your own boundary in form of a vector file (e.g. *.gpkg*, *.geojson*) with a valid polygon shape. 
+
+  2. As soon you are done click "**Generate boundary**" and your model boundary is established.
+
+For further details please refer to @fig-model_boundaries.
+
+::: {#fig-model_boundaries}
+
+{{< video ../../_static/videos/hydromt_fiat/Generate_Boundaries.mp4 >}}
+
+Create model boundaries using the "Draw Polygon" method.
+
+:::
+
+## Quickbuild
+If you set up a model in the United States and do not want to use your own data you can simply choose to work with the **quickbuild**. 
+
+The **quickbuild** downloads the data from different sources:  
+
+* **Exposure**: National Structure Inventory (NSI)
+* **Vulnerability curves**: FEMA’s Hazus Program
+* **Social vulnerability index**/**equity**: Census api
```

Comment:
Two things. 1 - I would just call this US census data, 2 - the index is not coming from the census. I would say 'socio-economic/equity data' here. Somewhere we need to describe the social vulnerability index feature, which by the way, I do not fully trust right now.